Pokemon Player/headers/Moving.h

```diff
@@ -0,0 +1,24 @@
+#ifndef MOVING_H
+#define MOVING_H
+
+#include <iostream>
+#include <fstream>
+#include "../headers/Control.h"
```

Comment:
You don't need any of these includes in the header file. You should move them to the .cpp file.